files/images/teamspeak_viewer/badges/cap_red_details.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64"><title>Cap</title><path d="M63.4,45.73c-2.94-2.44-5.89-4.87-8.81-7.33a3,3,0,0,0-1.87-.78,108.14,108.14,0,0,0-24.91.77,163.54,163.54,0,0,0-17.26,3.7c-.46.13-.46.23-.18.56,1.29,1.53,2.6,3,4,4.42a33,33,0,0,0,8.37,6.36,9.3,9.3,0,0,0,6.77.82,62,62,0,0,0,7.93-2.87,97.22,97.22,0,0,1,12.22-4.6,28.32,28.32,0,0,1,12.41-.89l1.6.27.06-.12Z" style="fill:#c2000b"/><path d="M13.14,38.37a135.44,135.44,0,0,1-1.48-16,35.28,35.28,0,0,1,.45-7.33,21.73,21.73,0,0,1,1.29-4.21l-.41.19a21,21,0,0,0-10.17,11A34.44,34.44,0,0,0,.29,34.32a35.92,35.92,0,0,0,.06,4.4c0,.35.11.44.47.31a7.3,7.3,0,0,1,2.12-.4A9.21,9.21,0,0,1,8,40a.82.82,0,0,0,.72.15c1.31-.38,2.63-.73,3.95-1.06C13.09,39,13.21,38.83,13.14,38.37Z" style="fill:#c2000b"/><path d="M46.43,25.41c.74,3,1.5,6,2,9.12,0,.33.15.47.52.49,1.23,0,2.45.13,3.67.22.36,0,.51,0,.41-.44-.17-.73-.26-1.47-.42-2.19a38.4,38.4,0,0,0-6.41-14.75,16.58,16.58,0,0,0-3-3.2c0,.08,0,.15,0,.22A82.6,82.6,0,0,1,46.43,25.41Z" style="fill:#c2000b"/><path d="M16.08,38.24c1.87-.47,3.76-.86,5.65-1.23a101.06,101.06,0,0,1,18.91-2.13h4.47c1,0,1,0,.79-1a139.36,139.36,0,0,0-3.64-14.55,30.13,30.13,0,0,0-3.45-7.86,1.13,1.13,0,0,0-.45-.47,26.13,26.13,0,0,0-8.82-2.71,31.94,31.94,0,0,0-12.07,1,1.2,1.2,0,0,0-.67.45,12.21,12.21,0,0,0-1.65,3,20.44,20.44,0,0,0-1.09,7.06,120.81,120.81,0,0,0,1.41,18C15.54,38.4,15.78,38.31,16.08,38.24Z" style="fill:#c2000b"/><path d="M37.89,25.12V23.53c0-.55-.55-.94-.82-1.09a.38.38,0,0,1-.18-.25,7.21,7.21,0,0,0-1.26-3,.2.2,0,0,1,0-.25h0a.21.21,0,0,0,0-.3,7.59,7.59,0,0,0-11.49,0,.21.21,0,0,0,0,.3l0,0a.19.19,0,0,1,0,.25A6.78,6.78,0,0,0,23,22.19a.34.34,0,0,1-.17.25A1.41,1.41,0,0,0,22,23.53v1.59a1.4,1.4,0,0,0,1.49,1.44s.47.1.47-.5v-2.6a6,6,0,1,1,12,0h0v2.6c0,.57.47.5.47.5A1.41,1.41,0,0,0,37.89,25.12Z" style="fill:#fff"/><path d="M35.65,27.65S35,30.82,30,31.3c0,0-2.14.17-.47.49s5.55.28,6.6-1.91a2.67,2.67,0,0,0,0-2.33S35.88,27.08,35.65,27.65Z" style="fill:#fff"/><path d="M26.92,27.62A1.85,1.85,0,0,1,25.23,26c-.34-1.57.18-2.24.57-2.34s1.59.1,3.35,2.24C30.82,27.92,26.92,27.62,26.92,27.62Z" style="fill:#fff"/><path d="M31.26,27.57s-1.44-.52-.25-1.38,3.23-1.54,3.65-.63a1.75,1.75,0,0,1-.87,2.09A3.79,3.79,0,0,1,31.26,27.57Z" style="fill:#fff"/></svg>
```

Comment:
Why are there no line breaks here?

Reply:
Rechecked with original source. The file is untouched from original.